myuw/templates/handlebars/teaching_resources.html

```diff
@@ -7,12 +7,12 @@ <h3 class="scheduleHeading">Teaching Resources</h3>
         <ul class="unstyled-list">
             <li><a href="http://www2.bookstore.washington.edu/textsys/TextReqLogin.taf?school=uwmain" rel="">Order Textbooks</a></li>
             <li><a href="http://www.lib.washington.edu/types/course/instructors/request" rel="">Request Course Reserves</a></li>
-            <li><a href="#" rel="">Request Course Packs</a></li>
+            <li><a href="http://f2.washington.edu/fm/c2/printing-copying/course-packs" rel="">Request Course Packs</a></li>
         </ul>
         <h4>Web Tools & Services</h4>
         <ul class="unstyled-list">
             <li><a href="http://canvas.uw.edu" rel="">Canvas</a></li>
-            <li><a href="https://catalyst.uw.edu" rel="">Catalyst Web Tools</a></li>
+            <li><a href="https://panopto.uw.edu" rel="">Panopto</a></li>
```

Comment:
I think "Panopto Lecture Capture" is a good way to go. It's also how we describe the service in IT Connect.